apps/code-examples/src/app/code-examples/forms/toggle-switch/inline-help/toggle-switch-demo.module.ts

```diff
@@ -1,12 +1,18 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
 import { SkyToggleSwitchModule } from '@skyux/forms';
 import { SkyHelpInlineModule } from '@skyux/indicators';
 
 import { ToggleSwitchDemoComponent } from './toggle-switch-demo.component';
 
 @NgModule({
-  imports: [CommonModule, SkyHelpInlineModule, SkyToggleSwitchModule],
+  imports: [
+    CommonModule,
+    SkyHelpInlineModule,
+    SkyToggleSwitchModule,
+    FormsModule,
```

Comment:
I removed the `FormsModule` and the code example still worked. Do we need this for a different reason?

Reply:
I was seeing an error on the preview site, but it looks like docstools provides `FormsModule` in StackBlitz so it wasn't broken there. I'm refactoring this to use a reactive form to simplify the example.